builds/e2e/templates/e2e-run.yaml

```diff
@@ -24,14 +24,15 @@ steps:
     {
       $filter += '&Category!=SingleNodeOnly'
       $filter += '&Category!=NestedEdgeAmqpOnly'
+      $filter += '&Category!=LegacyMqttRequired'
     
-      #We are disabling some additional tests. See  PBI #9171870. Once this is fixed we should run what's directly below.
-      #$filter += '&FullyQualifiedName!~Provisioning&FullyQualifiedName!~PlugAndPlay&FullyQualifiedName!~PriorityQueueModuleToHubMessages&FullyQualifiedName!~SasOutOfScope'
       $filter += '&FullyQualifiedName!~Provisioning&FullyQualifiedName!~PlugAndPlay&FullyQualifiedName!~PriorityQueueModuleToHubMessages&FullyQualifiedName!~SasOutOfScope&FullyQualifiedName!~X509ManualProvision&FullyQualifiedName!~AuthorizationPolicyUpdateTest&FullyQualifiedName!~AuthorizationPolicyExplicitPolicyTest'
```

Comment:
@vadim-kovalyov I chose to remove only the line directly below when we realized this PBI was completed. Therefore the AuthorizationPolicyExplicitTest is still filtered.

I tried to run the tests with both filter lines removed and the tests hung due to some Provisioning test.

Reply:
hm, I still wonder why AuthZ tests are filtered out. If Broker is enabled they should work in tested. I has been manually verified during bug bash. It is basically just a twin update...